include/deal.II/grid/grid_tools.h

```diff
@@ -1024,6 +1046,31 @@ namespace GridTools
   vertex_to_cell_map(const Triangulation<dim,spacedim> &triangulation);
 
   /**
+   * Returns a vector that contains a tensor for every vertex-cell combination
```

Comment:
Maybe add that it is a normalized vector. Otherwise it might be used in a wrong way by other users.